File: Java/src/1672_RichestCustomerWealth/RichestCustomerWealth.java
import java.util.ArrayList;
import java.util.Collections;

class Solution {
    public int maximumWealth(int[][] accounts) {
        ArrayList<Integer> ans = new ArrayList<>();
        for(int i = 0 ; i < accounts.length ; i++){
            int sum = 0;
            for(int j = 0 ; j < accounts[i].length ; j++){
                sum+= accounts[i][j];
            }
            ans.add(sum);
        }
        return  Collections.max(ans);
    }
}

File: golang/1672_RichestCustomerWealth/richestCustomerWealth.go
package richestcustomerwealth

func maximumWealth(accounts [][]int) int {
	ans := 0
	for i := 0; i < len(accounts); i++ {
		sum := 0
		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}
		if sum > ans {
			ans = sum
		}
	}

	return ans
}
